File: Lab_task(Manual_06).java
package javaapplication19;

import java.util.Scanner;

public class JavaApplication19 {
   
    public static boolean isPalindrome(String str){
        char[] str2 = new char[str.length()];
            for(int i=0 ; i<str.length() ; i++){
                str2[i] = str.charAt(str.length()-i-1);
            }
            
          return str.equals(new String(str2));
} 
    
    public static void main(String[] args) {
       Scanner input = new Scanner(System.in);
       
        System.out.println("Enter a string : ");
       String str = input.next();
       boolean val = isPalindrome(str);
       if(val==true){
           System.out.println("This is a Palindrome.");}
       else System.out.println("This is not a palindrome.");
    }
    
}

File: Lab_task(Manual_06).c

package javaapplication20;

import java.util.Scanner;

public class JavaApplication20 {

   public static boolean isEven(int num){
         return num % 2 == 0;
  
   }
    
   public static void generateEven(int min , int max){
          for(int i=min ; i<=max ; i++){
              if(isEven(i)){
                  System.out.print(i+" ");
              }
          }  
   
   }
    public static void main(String[] args) {
        
        Scanner input = new Scanner(System.in);
        System.out.println("Enter the minimum range: ");
        int min = input.nextInt();
        System.out.println("Enter the maximum range: ");
        int max =  input.nextInt();
        
        generateEven(min,max);
    }
    
}
